pkg/edgeview/dispatcher/wss-server.go

```diff
@@ -0,0 +1,265 @@
+package main
```

Comment:
Seems, we need copyright in the first lines.

Reply:
missed that. will add.